app/dashboards/customer_dashboard.rb

```diff
@@ -5,10 +5,15 @@ def title_attribute
     :name
   end
 
-  def index_attributes
+  def attributes
```

Comment:
actually, the naming scheme is a bit strange... how about `index_page_attributes` and `show_page_attributes`?